src/tdmlEditor/utilities/tdmlXmlUtils.ts

```diff
@@ -260,9 +260,36 @@ export async function getTestCaseDisplayData(
  * See the 'Invalid Append TDML - duplicate' and 'Append TDML Test Case' tests for examples
  */
 export async function appendTestCase(
-  xmlBuffer: string,
-  destinationBuffer: string
+  xmlFilePath: string,
+  destinationFilePath: string
+) {
+  return copyTestCase(xmlFilePath, destinationFilePath, true)
+}
+
+/*
+ * Copy and optionally append test case in one XML file into another file's test suite
+ *
+ * xmlFilePath: String containing the XML file path of a TDML file with
+ *   exactly one test case
```

Comment:
Concern: it doesn't really expect a TDML file with only one test case, does it? Then append would only work once and never again on the same file.

Reply:
This functionality need to be revisited in future release.